openid-4-verifiable-presentations-1_0.md

```diff
@@ -2677,6 +2744,22 @@ established by [@!RFC7515].
 * Change Controller: OpenID Foundation Artifact Binding Working Group - openid-specs-ab@lists.openid.net
 * Specification Document(s): (#verifier_attestation_jwt) of this specification
 
```

Comment:
I am not a DE on the JSON Web Signature and Encryption Header Parameters Registry so I can only speculate but if I were, I'd likely have questions...

Reply:
like? ...

Reply:
Well, the first question that would come to my mind is, "what the fuck is this?" but t think about it from the perspective that this will maybe eventually be put forwarded through various bureaucratic entities to be considered as a registered JWS header. JOSE is a general purpose thing that is *many* layers removed from this document and the DE's might not know or care about OpenID4VP or any of this stuff. What is `client_id` and what does it do and why does it need or deserve to be general reserved JOSE header? `This header contains a Client Identifier.` and a pointer to RFC6749 are not, in my narrow view as the DE for a different registry, helpful at all.